.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will build a .NET project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
+
+name: .NET
+
+on:
+  push:
+    branches: ["dui3/alpha", "dui3/ci/*"]
+  pull_request:
+    branches: ["dui3/alpha"]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 7.x.x
+      - name: Add msbuild to PATH
+        uses: microsoft/setup-msbuild@v2
+      # - name: Restore tools
+      #   run: dotnet tool restore
+      # - name: Code formatting check
+      #   run: dotnet csharpier --check .
+      - name: Restore projects
+        run: dotnet restore DUI3-DX.slnf
+      - name: Build
+        run: msbuild DUI3-DX.slnf /p:Configuration=Release /p:IsDesktopBuild=false
```

Comment:
not sure if this helps but using bullseye and C# instead of msbuild is what I do: https://github.com/adamralph/bullseye

Reply:
Nice! I've been eyeing some build systems for a while but didn't know about this one. Adding it to my "must look into" list!

Reply:
I like that the build is the same language as the project and not XML.

I used to use (and contribute) to Cake but thought it was overblown.  Found bullseye and used it ever since dotnet got simple and fast